install.sh

```diff
@@ -63,10 +67,12 @@ mv "${tmpLog}" /
 
 if [[ $USE_CUSTOM_BRANCHES == true ]] ; then
     ln -s /bin/true /usr/local/bin/service
-    echo "$CORE_TAG" | tee /etc/pihole/ftlbranch
+    ln -s /bin/true /usr/local/bin/update-rc.d
     echo y | bash -x pihole checkout core $CORE_TAG
     echo y | bash -x pihole checkout web $CORE_TAG
+    echo y | bash -x pihole checkout ftl ${CORE_TAG/v/}
```

Comment:
New way to checkout dev copies of FTL added as I was pre-release testing.  Noticed no 'v' was in this tag where as the others weren't so I'm search/replacing it in this bash variable.